helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -226,6 +230,10 @@ splunk_hec/platform_metrics:
     {{- if .Values.splunkPlatform.caFile }}
     ca_file: /otel/etc/splunk_platform_hec_ca_file
     {{- end }}
+  retry_on_failure:
+    {{- toYaml .Values.splunkPlatform.retry_on_failure | nindent 4 }}
```

Comment:
the previous convention appears to be camel case so I'd suggest staying with that. We may want to treat all fields individually like is done w/ tls above to avoid a direct map injection method?

Reply:
Makes sense.